include/seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm.hpp

```diff
@@ -141,9 +141,18 @@ class pairwise_alignment_algorithm : protected policies_t...
     {
         using std::get;
 
+        thread_local score_matrix_single_column<int32_t> alignment_matrix{};
+        coordinate_matrix<uint32_t> index_matrix{};
+
         for (auto && [sequence_pair, idx] : indexed_sequence_pairs)
         {
-            compute_matrix(get<0>(sequence_pair), get<1>(sequence_pair));
+            size_t number_of_columns = std::ranges::distance(get<0>(sequence_pair)) + 1;
+            size_t number_of_rows = std::ranges::distance(get<1>(sequence_pair)) + 1;
```

Comment:
```suggestion
            size_t const number_of_columns = std::ranges::distance(get<0>(sequence_pair)) + 1;
            size_t const number_of_rows = std::ranges::distance(get<1>(sequence_pair)) + 1;
```

Reply:
this is in the final version already fixed.